planner/core/resolve_indices.go

```diff
@@ -296,7 +296,7 @@ func (p *PhysicalIndexLookUpReader) ResolveIndices() (err error) {
 		p.ExtraHandleCol = newCol.(*expression.Column)
 	}
 	for i, commonHandleCol := range p.CommonHandleCols {
-		newCol, err := commonHandleCol.ResolveIndices(p.indexPlan.Schema())
+		newCol, err := commonHandleCol.ResolveIndices(p.tablePlan.Schema())
```

Comment:
Does `tablePlan.Schema()` necessarily contain common handle columns?

Reply:
Yes. we never prune handle columns